src/AlgoliaSearch.js

```diff
@@ -85,6 +85,18 @@ AlgoliaSearch.prototype.copyIndex = function(srcIndexName, dstIndexName, scopeOr
   });
 };
 
+AlgoliaSearch.prototype.copySettings = function(srcIndexName, dstIndexName, callback) {
```

Comment:
It's not defined on the CTS yet, but IMO we should add tests on those methods as-well.